tests/jobs/test_backfill_job.py

```diff
@@ -763,15 +778,13 @@ def test_backfill_skip_active_scheduled_dagrun(self, dag_maker, caplog):
         # since DAG backfill is skipped, task state should be none
         assert ti.state == State.NONE
 
-    def test_backfill_rerun_failed_tasks(self, dag_maker):
+    def test_backfill_rerun_failed_tasks(self, dag_maker, mock_executor):
         dag = self._get_dummy_dag(
```

Comment:
I know this wasn't you, but pretty sure we have "dummy" on our list of words to avoid.  We should maybe update that method name at some point.  I thought we had a CI check for "unacceptable words"

Reply:
We do - but some things are excluded  - I think that's one of them (and yes we should change it)

Reply:
Agreed, but I think maybe that should be a separate PR!